domains/devcomp.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Cool looking redirection for my site.",
+  "repo": "https://github.com/CompeyDev",
+  "owner": {
+    "username": "CompeyDev",
+    "email": "hi@devcomp.tk"
+  },
+{
```

Comment:
```suggestion

```
this is useless + doesnt match things. please refer to [the docs](https://github.com/is-a-dev/register/blob/main/docs/domains-json.md) for more info.

Reply:
also delete line 8 please